lib/relation-definition.js

```diff
@@ -1369,6 +1411,20 @@ BelongsTo.prototype.related = function (refresh, params) {
   }
 };
 
+
+/**
+ * Define a Promise-based method for the belongsTo relation itself
+ * - order.customer.get(refresh): Load the target model instance asynchronously
+ *
+ * @param refresh
+ * @returns {Promise}
+ */
+BelongsTo.prototype.get = function (refresh) {
+  var cb = utils.createPromiseCallback();
+  return this.related(refresh, cb);
+}
```

Comment:
+1 to allow callback. Isn't `order.user(cb)` the same as `order.user.get(cb)`? I wonder what's the motivation to add a `get`.